app/Services/GetPhpVersion.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace App\Services;
+
+use Illuminate\Support\Facades\Process;
+
+class GetPhpVersion
+{
+    /**
+     * Determine PHP Version
+     * First Case: Can't determine version through command line; use $default version || TESTCASE: $resultArr = [];
+     * Second Case: Version is below 7.4; use version 7.4 || TESTCASE: $resultArr = [0,"7.3"];
+     * Third Case: Version received is okay; use version received || TESTCASE: $resultArr = [0,"7.6"]; $resultArr = [0,"8.2"];
+     */
+    public function get( \App\Commands\Launch $launch ): string
+    {
+        // Default Values
+        $defaultVersion = "8.0";
+        $regex = '/PHP ([0-9]+\.[0-9]+)\.[0-9]/';
+
+        // Get Version Output
+        $result = Process::run( 'php -v' );
+
+        // Extract Version from Output
+        $resultArr = [];
+        preg_match( $regex, $result->output(), $resultArr );
+        
+        // Determine Version Extracted
+        if( count($resultArr) > 1 )
+        {
+            // Below 7.4 versions dont have container, use 7.4 instead
+            if( strpos($resultArr[1][0],'7')!==false )
```

Comment:
not sure why this `if` is used, cant you just cast the version to a float and check if it's smaller than 7.4 like you do in Line 33-34?

Reply:
Check out [`version_compare()`](https://www.php.net/manual/en/function.version-compare.php), it might be more handy than casting strings to floats.

Reply:
> Check out [`version_compare()`](https://www.php.net/manual/en/function.version-compare.php), it might be more handy than casting strings to floats.

Will check!

Reply:
oh actually, we're doing this mega wrong 😂, I didn't think about this.

We're already running PHP for these artisan commands, so we can use [`phpversion()`](https://www.php.net/manual/en/function.phpversion.php)